shared/teams/role-picker/controlled-container.js

```diff
@@ -23,8 +23,16 @@ export type ControlledRolePickerProps = {
   showNotificationCheckbox?: boolean,
   sendNotificationChecked?: boolean,
   styleCover?: Object,
+  teamname: string,
 }
 
+type OwnProps = RouteProps<
+  {
+    ...ControlledRolePickerProps,
```

Comment:
Why like this instead of just `ControlledRolePickerProps` ?

Reply:
i wasn't sure if more stuff would have to be hoisted in there or not while doing it